gcloud/datastore/test_key.py

```diff
@@ -17,8 +17,9 @@
 
 class TestKey(unittest2.TestCase):
 
+    _DEFAULT_DATASET = 'DATASET'
+
     def setUp(self):
```

Comment:
Is this needed if `_monkeyDatasetID` is used throughout?

ISTM for maintenance reasons we're better off using an explicit `dataset_id` in the `Key` constructor for most of these tests rather than proliferating the use of `with self._monkeyDatasetID(...):`.